demo/index.md

```diff
@@ -205,6 +205,67 @@ It is possible to automatically sort the table once you create a Tablesort insta
 </tbody>
 </table>
 
+### Sorting by column keys
+
+Sometimes, tables can have more complex column structures, especially when using colspans. In these cases, you can
+explicitly connect a header to the cells in each row that it should sort by, by using the `data-sort-column-key`
+attribute.
+
+This example sorts products by price, even though the prices are not in the same column as their header.
+
+{% highlight html %}<table id='column-keys' class='sort'>
```

Comment:
Maybe leave out the `id` attribute here? duplicate ids can be finicky even though this is wrapped in highlight tags